src/database/schema/ChannelListMemberSchema.ts

```diff
@@ -173,6 +175,8 @@ class ChannelListMemberSchema implements BaseDbSchema {
 
   static fromMessageAnonymousChatAPI = ChannelListMemberSchema.fromWebsocketObject;
 
+  //! TODO:
+  //! UPDATE ANONYMOUS DATA MEMBER FOR SIGNED CHAT
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, this comment suggests that there is a need to update the anonymous data member for signed chat. But no changes have been made in this hunk. Please make sure to implement the necessary changes or remove the comment if it's not relevant.
```diff
-  //! TODO:
-  //! UPDATE ANONYMOUS DATA MEMBER FOR SIGNED CHAT
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->